proposal/more-math-functions.md

```diff
@@ -0,0 +1,445 @@
+# Math Module
```

Comment:
Can you format the intro more like [other proposals](https://github.com/sass/sass/blob/master/proposal/import-configuration.md), with a short proposal name, draft number, and link to the GitHub issue?

Reply:
Done.